.github/workflows/java_binding_publish.yml

```diff
@@ -0,0 +1,136 @@
+name: Create Java release
+
+on: workflow_dispatch
+
+jobs:
+  create-releases:
+    name: Create Java release
+    if: ${{ ! github.event.schedule }}
```

Comment:
What does this mean ?

Reply:
Oh I guess we can remove this line (copied by mistake from some other workflow I guess) since its for scheduling of the job https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Reply:
fixed it